CondFormats/GeometryObjects/src/HcalParameters.cc

```diff
@@ -0,0 +1,16 @@
+#include "CondFormats/GeometryObjects/interface/HcalParameters.h"
+
+double HcalParameters::getEtaHF(const unsigned int i) const {
+  unsigned int k = rTable.size()-i-1;
+  double eta = -log(tan(0.5*atan(rTable[k]/gparHF[4])));
+  return eta;
+}
+
+std::vector<double> HcalParameters::getEtaTableHF() const {
+
```

Comment:
Have you included this in the new PR? It will certainly ease future usage.
________________________________
From: Ianna Osborne [notifications@github.com]
Sent: 10 July 2015 09:36
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx18 add Sim/Rec numbering records (#10077)


In CondFormats/GeometryObjects/src/HcalParameters.cc<https://github.com/cms-sw/cmssw/pull/10077#discussion_r34333124>:

> @@ -0,0 +1,16 @@
> +#include "CondFormats/GeometryObjects/interface/HcalParameters.h"
> +
> +double HcalParameters::getEtaHF(const unsigned int i) const {
> +  unsigned int k = rTable.size()-i-1;
> +  double eta = -log(tan(0.5*atan(rTable[k]/gparHF[4])));
> +  return eta;
> +}
> +
> +std::vector<double> HcalParameters::getEtaTableHF() const {
> +


@bsunanda<https://github.com/bsunanda> - I suggest we update the schema to include pre-calculated eta table vector for HF.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/10077/files#r34333124>.